framework/test/mocha/functional/ws/transport/transport.js

```diff
@@ -32,7 +32,7 @@ describe('WS transport', () => {
 		let p2p;
 
 		function postTransaction(transactionToPost) {
-			p2p.send({
+			return p2p.send({
```

Comment:
Do we really need this return? I actually don't like the structure of the entire describe. I suggest to clean it:
- Its `beforeEach` is misleading
- Remove the use of done.
- The inner `describe` block is not necessary
- I would remove `postTransaction()` to ease the understanding of every test or move it to a more general handler. If you don't agree, we should follow this pattern later on and do the same thing when sending.

Reply:
This PR is not intended to refactor the test.
All of the functional tests will be recovered in jest, so not needed to rewrite for this patch imo